src/prod.py

```diff
@@ -75,11 +75,29 @@ def run(self, args, battery_charge, horizon):
         cwd = os.getcwd()
         install_path = getInstallPath()
         os.chdir(install_path)
-        
+
         hashrate_bonus = 0
-        if args.in_data and args.out_dir:
+        try:
+            a = POW_Coin(kraken.coin.XMR)
+            try:
+                a.historical_diff(timestamp=datetime.datetime.now().timestamp())
```

Comment:
A slightly cleaner version of this would be:

`a.historical_diff(height=a.height)`

That way it doesn't need to do all the timestamp-height guessing, but will directly fetch until the last known height.

Reply:
Cool. Will do.